ui/oopsyraidsy/data/04-sb/raid/o2n.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+// O2N - Deltascape 2.0 Normal
+[{
+  zoneRegex: {
+    en: /^Deltascape \(V2\.0\)$/,
+  },
+  damageWarn: {
+    'O2N Main Quake': '24A5', // Non-telegraphed circle AoE, Fleshy Member
+    'O2N Erosion': '2590', // Small circle AoEs, Fleshy Member
+  },
+  triggers: [
+    {
+      // We could try to separate out the mistake that led to the player being petrified.
+      // However, it's Normal mode, why overthink it?
+      id: 'O2N Petrification',
+      gainsEffectRegex: gLang.kEffect.Petrification,
+      // The user might get hit by another petrifying ability before the effect ends.
+      // There's no point in notifying for that.
+      suppressSeconds: 10,
+      mistake: function(e) {
+        return { type: 'warn', name: e.targetName, text: e.effectName };
+      },
+    },
+    {
+      // Instant tank cleave
+      id: 'O2N Paranormal Wave',
+      damageRegex: '250E',
+      condition: function(e) {
+        // Double taps only.
+        return e.type != '15';
```

Comment:
I really do think it'd be nice to have a doubleTapWarn helper for oopsy.  That seems to be the most common manual trigger these days.